documentation-website/Writerside/snippets/exposed-dao/src/main/kotlin/org/example/entities/UserEntityWithOverride.kt

```diff
@@ -0,0 +1,17 @@
+package org.example.entities
+
+import org.example.tables.UsersTable
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+
+class UserEntityWithOverride(id: EntityID<Int>) : IntEntity(id) {
+    companion object : IntEntityClass<User>(UsersTable)
+
+    var name by UsersTable.name
```

Comment:
Unless I just can't find it, same issue as above with `User`:
`companion object : IntEntityClass<UserEntityWithOverride>(UsersTable)`

This table isn't being used with any operations, but if it was there might be problems because `UsersTable` defines a `cityId` that doesn't have a corresponding property here. So either `UsersTable.cityId` would need to be nullable (via `optReference`) or a property needs to be defined here too:
`var city by CityEntity referencedOn UsersTable.cityId`

Reply:
Fixed! Also added this to `UserEntity` as it was missing there too.